lib/Api.js

```diff
@@ -225,6 +225,23 @@ class Api {
       });
   }
 
+  updateObject(req, res, next) {
+    let self = this;
```

Comment:
@kleinmp @dzink - Your default variable declaration should be `const` instead of `let`. In JS, `const` is still bound by function scope (really any block scope - e.g. var is not bound by if statements but const and let are), it can't leak out of this scope. It does not mean the same thing in JS as it does in PHP or other languages. 

The reason to use `const` by default is to ensure the variable binding remains the same. `let` can be redefined within a scope, whereas `const` cannot. Defaulting to `const` requires you to think about if you want this variable to be reboundable to something different. 95% of the time, you never want to reuse the same variable name within one scope.